exporters/otlp-http/metrics/src/test/java/io/opentelemetry/exporter/otlp/http/metrics/OtlpHttpMetricExporterTest.java

```diff
@@ -94,6 +96,43 @@ void setup() {
                 HELD_CERTIFICATE.certificatePem().getBytes(StandardCharsets.UTF_8));
   }
 
+  @Test
+  @SuppressWarnings("PreferJavaTimeOverload")
+  void validConfig() {
```

Comment:
Did this in another place and I think its useful to show explicitly config we expect to support.

One thing that's strange is that we're too permissive with the values we accept for `setEndpoint`.  We only confirm that the values are valid URIs, and that the scheme is `http` or `https`. The test cases on 111-117 are a small subset of what is accepts. The autoconfigure module now does additional validation, but arguably that validation should be moved to the exporter builders instead.